src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,101 @@
 package core.basesyntax;
 
+import java.io.*;
+
 public class WorkWithFile {
+    private static final String BUY = "buy";
+    private static final String SUPPLY = "supply";
+    private static final int ACTION_INDEX = 0;
+    private static final int MOUNT_INDEX = 1;
```

Comment:
dont invent your own names, when they already exist
```suggestion
    private static final int OPERATION_AMOUNT_INDEX = 1;
```